lib/routes/account.js

```diff
@@ -390,6 +395,26 @@ module.exports = function (
             )
         }
 
+        function checkNumberOfActiveSessions () {
+          return db.sessions(emailRecord.uid)
```

Comment:
>  adds a db call for a user's sessions

This call was already present for the majority of requests, further down in the `emitSyncLoginEvent` function.  That's also why we store the result into the `sessions` variable, so that this function can use it later.